usaspending_api/references/management/commands/load_gtas.py

```diff
@@ -40,7 +57,7 @@ def handle(self, *args, **options):
 
         logger.info("Inserting GTAS total obligations records into website")
         total_obligation_objs = [GTASSF133Balances(**values) for values in total_obligation_values]
-        GTASSF133Balances.objects.bulk_create(total_obligation_objs)
+        GTASSF133Balances.objects.bulk_create(total_obligation_objs, batch_size=100000)
```

Comment:
How did you determine 100K record per batch was optimal? Was it memory utilization or something else which prompted this?

Reply:
This was originally added to ensure that the command could be run locally with a Postgres instance running in Docker. After upgrading to Docker 3.0.1, I have determined it is no longer necessary. I have removed the `batch_size` argument from the PR